webpack.config.js

```diff
@@ -28,5 +28,9 @@ module.exports = {
       },
     ]
   },
-  plugins: [HTMLWebpackPlugin, HotReloader]
+  plugins: [HTMLWebpackPlugin, HotReloader],
+  devServer: {
+    contentBase: __dirname + '/dist',
+    hot: true,
+  }
```

Comment:
This is the `webpack-dev-server` config.  It is a separate module and separate from the `webpack` config.  Webpack will handle merging this section of the config with the cli options:

>Configuration under devServer in webpack.config.js are merged with the CLI options. For options under devServer see next section.
>https://webpack.github.io/docs/webpack-dev-server.html#webpack-dev-server-cli